packages-next/auth/src/gql/getPasswordResetSchema.ts

```diff
@@ -125,10 +126,12 @@ export function getPasswordResetSchema({
           const result = await validateAuthToken(
             'passwordReset',
             list,
+            listKey,
             identityField,
             protectIdentities,
             passwordResetLink.tokensValidForMins,
-            args
+            args,
+            ctx
```

Comment:
Is this inconsistent with the rename of the `ctx` prop to `context` below in `redeemAuthToken`?

Reply:
It is, this is going to get renamed in a future PR (there are a few other places in this same category which need to be cleaned up across the board at this level).